core/src/main/scala/org/apache/spark/ui/SparkUI.scala

```diff
@@ -135,7 +133,7 @@ private[spark] class SparkUI private (
 private[spark] abstract class SparkUITab(parent: SparkUI, prefix: String)
   extends WebUITab(parent, prefix) {
 
-  def appName: String = parent.getAppName
+  def appName: String = parent.appName
```

Comment:
I might have missed why we could remove it. Mind refreshing my memory @srowen (sorry getting too old :))